src/read_pool.rs

```diff
@@ -144,6 +150,22 @@ impl ReadPoolHandle {
             rx.map_err(ReadPoolError::from).await
         }
     }
+
+    pub fn is_busy(&self) -> bool {
```

Comment:
Could we use something like "pending_tasks_per_worker" to avoid passing `MAX_BATCH_GET_REQUEST_COUNT` all the way in?

Reply:
Yes, I have rewrite this code.